recipe/meta.yaml

```diff
@@ -6,37 +6,42 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: http://www.shoup.net/ntl/ntl-{{ version }}.tar.gz
-  sha256: 84ba3145abf8d5f3be6832a14c60b3368eb920719ee96e5774587e71ecd66e9d
+  - url: http://www.shoup.net/ntl/ntl-{{ version }}.tar.gz
+    sha256: 84ba3145abf8d5f3be6832a14c60b3368eb920719ee96e5774587e71ecd66e9d
+    patches:
+      - cmake.patch   # [win]
+      - config.patch
+      - mach_desc.h.patch  # [win]
 
 build:
-  number: 1000
-  skip: true        # [win]
+  number: 1002
+  skip: True    # [win and vc<14]
   track_features:
     - ntl
   run_exports:
     - ntl {{ version }}
 
 requirements:
   build:
-    - libtool
-    - automake
+    - libtool               # [not win]
+    - automake              # [not win]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - cmake                 # [win]
+    - pthreads-win32        # [win]
```

Comment:
This needs to be in host and run. Not build

Reply:
True, thanks.